app/views/hyrax/base/_form_relationships.html.erb

```diff
@@ -1,10 +1,11 @@
-<% if Flipflop.assign_admin_set? %>
-  <%# TODO: consider `Hyrax::AdminSetOptionsPresenter.select_options_for(controller: controller)` instead %>
-  <%= f.input :admin_set_id, as: :select,
-      selected: ScholarsArchive::AdminSetSelectService.select(f.object.model.class.to_s, Hyrax::AdminSetOptionsPresenter.new(Hyrax::AdminSetService.new(controller)).select_options),
-      include_blank: true,
-      collection: Hyrax::AdminSetOptionsPresenter.new(Hyrax::AdminSetService.new(controller)).select_options,
-      input_html: { class: 'form-control' } %>
+<div class=<%= current_user.admin? ? "" : "hidden" %>>
+  <% if Flipflop.assign_admin_set? %>
+    <%# TODO: consider `Hyrax::AdminSetOptionsPresenter.select_options_for(controller: controller)` instead %>
+    <%= f.input :admin_set_id, as: :select,
+    selected: ScholarsArchive::AdminSetSelectService.select(f.object.model.class.to_s, Hyrax::AdminSetOptionsPresenter.new(Hyrax::AdminSetService.new(controller)).select_options),
```

Comment:
I think the formatting of this `f.input` needs to be fixed with some left indent